app/src/main/java/ch/sdp/vibester/activity/ScoreBoardActivity.kt

```diff
@@ -37,8 +60,8 @@ class ScoreBoardActivity : AppCompatActivity() {
         }
     }
 
-    private fun loadPlayers() {
-        dbRef.orderByChild("ranking")
+    private fun loadPlayersSortedBy(order: String) {
```

Comment:
UsersRepo contains all the firebase calls connected with "users", should this function be moved there?

Reply:
I see what you mean, however, It's more logical for me to put it here because this function is only used locally in scoreboard activity to load the data into the scoreboard.